cloudant-client/src/main/java/com/cloudant/client/api/views/Key.java

```diff
@@ -79,7 +79,7 @@
         /**
          * Type constant for
          * <a target="_blank"
-         * href="https://docs.cloudant.com/creating_views.html#map-function-examples">
+         * href="https://console.bluemix.net/docs/services/Cloudant/api/creating_views.html#map-function-examples">
```

Comment:
Similarly to the "advanced" page problem this link works from the TOC on the creating_views page, but not when used directly.